sagemaker-inference-recommender/python-sdk-inference-recommender/inference-recommender-with-python-sdk.ipynb

```diff
@@ -0,0 +1,654 @@
+{
```

Comment:
<p>Reduce all "Step ..." headings to H2, H1 is reserved for the title of the notebook only. You may need to step down all other headers by one as well</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>